sql/sqlwire/sql_api.proto

```diff
@@ -84,21 +105,16 @@ message Result {
 message SQLRequest {
   // Request header.
   optional SQLRequestHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
-  // SQL commands/queries to be serially executed by the server.
-  message Cmd {
```

Comment:
We previously discussed having a single SQL string so as to avoid doing any parsing on the client. I don't recall there being an significant objection to doing so.